tests/test_lstm_policy.py

```diff
@@ -63,3 +95,37 @@ def test_lstm_policy(model_class, policy):
     finally:
         if os.path.exists("./test_model"):
             os.remove("./test_model")
+
+
+@pytest.mark.expensive
+def test_lstm_train():
+    """Test that LSTM models are able to achieve >=450 (out of 500) reward on CartPoleNoVelEnv.
+
+    This environment requires memory to perform well in."""
+    def make_env(i):
+        env = CartPoleNoVelEnv()
+        env = TimeLimit(env, max_episode_steps=500)
+        env = bench.Monitor(env, None, allow_early_resets=True)
+        env.seed(i)
+        return env
+
+    env = SubprocVecEnv([lambda: make_env(i) for i in range(NUM_ENVS)])
+    env = VecNormalize(env)
+    model = PPO2(MlpLstmPolicy, env, n_steps=128, nminibatches=NUM_ENVS, lam=0.95, gamma=0.99,
+                 noptepochs=10, ent_coef=0.0, learning_rate=3e-4, cliprange=0.2, verbose=1)
+
+    eprewmeans = []
+    def reward_callback(local, _):
+        nonlocal eprewmeans
+        eprewmeans.append(safe_mean([ep_info['r'] for ep_info in local['ep_info_buf']]))
+
+    model.learn(total_timesteps=100000, seed=0, callback=reward_callback)
+
+    # Maximum episode reward is 500.
+    # In CartPole-v1, a non-recurrent policy can easily get >= 450.
+    # In CartPoleNoVelEnv, a non-recurrent policy doesn't get more than ~50.
+    # LSTM policies can reach above 400, but it varies a lot between runs; consistently get >=150.
+    # See PR #244 for more detailed benchmarks.
+
+    average_reward = sum(eprewmeans[-4:]) / 4
```

Comment:
why only the last 4 episodes?
Use maybe a variable for that, it would be easier to read

Reply:
Made it a constant. 4 was somewhat arbitrary, I've increased it to 10 now. Don't want it to be too large since we'll only have ~200 episodes in training.